Microsoft.Toolkit.Uwp.UI.Controls/MarkdownTextBlock/MarkdownTextBlock.xaml

```diff
@@ -1,16 +1,29 @@
 ﻿<ResourceDictionary xmlns="http://schemas.microsoft.com/winfx/2006/xaml/presentation"
                     xmlns:x="http://schemas.microsoft.com/winfx/2006/xaml"
                     xmlns:controls="using:Microsoft.Toolkit.Uwp.UI.Controls">
+
+    <ResourceDictionary.ThemeDictionaries>
+        <ResourceDictionary x:Key="Default"/>
+        <ResourceDictionary x:Key="Light">
+            <SolidColorBrush x:Key="MarkdownBorderBrush" Color="#FFBEBEBE" />
```

Comment:
I was using the previously used colors in that control. Should I change it to default theme resources?